src/navigation/breadcrumbs/breadcrumbs.jsx

```diff
@@ -200,15 +233,26 @@ function Breadcrumbs(props) {
         `${BEM_NAVIGATION_BREADCRUMBS}--breadcrumb_title_typography`,
     );
 
+    const shouldUseStaticBreadcrumbs = Array.isArray(staticCrumbs);
+    const shouldUseDynamicBreadcrumbs = !!(staticCrumbs.length === 0 && router);
```

Comment:
Could we use some function/expression on `router` that returns a proper Boolean to avoid `!!`?
e.g.
```JavaScript
const shouldUseDynamicBreadcrumbs = staticCrumbs.length === 0 && !isNil(router);
```

Reply:
done.

Reply:
beauty...thank you sir!  Resolving.